spec/support/acts_as_tenant.rb

```diff
@@ -9,10 +9,12 @@ def with_tenant(tenant, &proc)
     context "with tenant #{tenant.inspect}" do |*params|
       before(:each) do
         ActsAsTenant.current_tenant = send(tenant)
+        @old_host = Capybara.app_host
```

Comment:
If not then <code>with_tenant</code> is not work, it's actually not inside that tenant, but in current host's tenant.  And you always need to put a <code>headers['host']</code>  before this function in your specs.